### PR TITLE
Facia styling fixes

### PR DESCRIPTION
**Pull request by sammorrisdesign-zz**
Sorry, didn't know what it call this branch. It fixes these things though;

Before:
screen shot 2015-05-11 at 17 02 59

After:
screen shot 2015-05-11 at 17 02 38

Before:
screen shot 2015-05-11 at 17 21 55

After:
screen shot 2015-05-11 at 17 22 33

Before (the lack of pulse, not the last updated blocks):
screen shot 2015-05-11 at 17 43 08

After (just adding the pulse):
screen shot 2015-05-11 at 17 42 53

### CODE REVIEW

File: static/src/inline-svgs/icon/star.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg width="14" height="13"><path d="M0 5.2l3.7 2.8-1.4 4.6.5.4 3.7-2.8 3.7 2.8.5-.4-1.4-4.6 3.7-2.8-.2-.6h-4.6l-1.4-4.6h-.6l-1.4 4.6h-4.6l-.2.6z"/></svg>
+<svg width="14" height="13" viewBox="0 0 14 13"><path d="M0 5.2l3.7 2.8-1.4 4.6.5.4 3.7-2.8 3.7 2.8.5-.4-1.4-4.6 3.7-2.8-.2-.6h-4.6l-1.4-4.6h-.6l-1.4 4.6h-4.6l-.2.6z"/></svg>
```
Comment (sndrs) [inline static/src/inline-svgs/icon/star.svg]: can the height/width go now?

**Comment by sndrs** [comment]
is the leading on those stars a bit tight?

**Comment by sndrs** [comment]
:+1: apart from comments

File: static/src/inline-svgs/icon/star.svg (hunk 1 above)
Comment (sammorrisdesign) [inline static/src/inline-svgs/icon/star.svg]: It probably could, but not without refactoring all the stars across the site. So I'm going to suggest that's part of another PR instead...

**Pull request merged**